odoo/tools/xml_utils.py

```diff
@@ -289,3 +289,8 @@ def validate_xml_from_attachment(env, xml_content, xsd_name, reload_files_functi
             _check_with_xsd(xml_content, xsd_name, env)
         except FileNotFoundError:
             _logger.warning("The XSD file(s) could not be found, even after a reload")
+
+
+def find_xml_value(xpath, xml_element, namespaces=None):
```

Comment:
Observations:
- Rename the function to something like `get_text`, `inner_text`.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText
- Rename the `element` variable to `elements`.
- Consider joining text from all the found `elements` like
```py
     return " ".join([element.text for element in elements]) if elements else None
```
I say this because `l10n_it_edi` would make intensive use of this.

Reply:
By convention, I would like to keep `find_value` & `find_all_value`. That way, you will be able to concatenate the values using `" ".join(find_all_value(...))`

Reply:
`find_all_values`

Reply:
I agree that we could also add ```find_all_values``` when it becomes relevant.